test/game-test.js

```diff
@@ -68,6 +68,17 @@ it('should only pick a survey that has not been used', () => {
   { answer: 'Dryer Sheets', respondents: 6, surveyId: 8}])
 })
 
+it('should call something', () => {
```

Comment:
Get more specific with this `it` block title -- what should be called? Remember, the tests should be written clearly enough that they can serve as documentation